pkg/parser/parser_test.go

```diff
@@ -1073,6 +1073,21 @@ AAAAAAAAAAAA5gm5Mg==
 		{"show table t1 partition (p0,p1) distributions", true, "SHOW TABLE `t1` PARTITION(`p0`, `p1`) DISTRIBUTIONS"},
 		{"show table t1 partition (p0,p1) distributions where a=1", true, "SHOW TABLE `t1` PARTITION(`p0`, `p1`) DISTRIBUTIONS WHERE `a`=1"},
 
+		// for distribute table
```

Comment:
should we add case for rule=peer, engine=tiflash? 
and also cases for the table without partition.

Reply:
done